product/models.py

```diff
@@ -6,7 +6,7 @@ class Product(models.Model):
     description             = models.CharField(max_length=2000, default="")
     note                    = models.CharField(max_length=3000, null=True)
     volume                  = models.CharField(max_length=50, default="")
-    price                   = models.DecimalField(max_digits=10, decimal_places=4, default=0)
+    price                   = models.DecimalField(max_digits=10, decimal_places=2, default=0)
```

Comment:
Product 클래스 내부 코드 간격이 너무 넓습니다. `description` 속성 기준으로 간격 맞춰주세요
```suggestion
    price          = models.DecimalField(max_digits=10, decimal_places=2, default=0)
```